pkg/cmd/provisioning/nutanix/create_shared_secrets.go

```diff
@@ -180,3 +257,20 @@ func initEnvForCreateCmd(cmd *cobra.Command, args []string) {
 		log.Fatalf("failed to create manifests directory at %s", manifestsDir)
 	}
 }
+
+// GetHomePath return home directory according to the system.
+// if the environmental virables does not exist, will return empty
```

Comment:
```suggestion
// GetHomePath returns the home directory according to the system.
// if the environmental variable does not exist, will return empty
```

Reply:
GetHomePath function was old code that has been removed.